src/cluster.py

```diff
@@ -241,6 +256,38 @@ def get_primary(self, unit_name_pattern=False) -> str:
                             primary = "/".join(primary.rsplit("-", 1))
                         return primary
 
+    def get_standby_leader(
+        self, unit_name_pattern=False, check_whether_is_running: bool = False
+    ) -> str:
```

Comment:
None if standby leader not found
```suggestion
    ) -> Optional[str]:
```

Reply:
It was updated on https://github.com/canonical/postgresql-operator/pull/452/commits/e918e4dfb7410cd9d4d48b5c8f1bc93198c0918a.